src/Slider/Ticks.tsx

```diff
@@ -20,16 +20,22 @@ interface Props {
 
 const Ticks: FC<Props> = ({
     valueRange,
-    range: { min, max, decimals = 0 },
+    range: { min, max, decimals = 0, step = null, explicitRange = [] },
```

Comment:
Do we need to have a default value for `step` and `explicitRange`, can't we treat `undefined` and `null` as equals, in the case for `step` at least.